tests/pytest/tests.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 [[ $VERBOSE == 1 ]] && set -x
-[[ $IGNERR == 1 ]] || set -e
+# [[ $IGNERR == 1 ]] || set -e
```

Comment:
@rafie revert?

Reply:
Intentional. `shibumi/defs` installs error handler. However, the `set -x` need to be commented out as it creates too much noise when going `VERBOSE=1`.